alibi_detect/utils/schemas.py

```diff
@@ -0,0 +1,431 @@
+# mypy: ignore-errors
```

Comment:
Casting a broad net here?

Reply:
Yes a very broad net! All the mypy errors in this file are actually `assignment` errors, so we could be more specific. 

But I've struggled to actually fix them. Could do with input wrt to this. The errors are all similar to the following:
```
alibi_detect/utils/schemas.py:102: error: Incompatible types in assignment (expression has type "Callable[..., Any]", base class "PreprocessConfig" defined the type as "str")  [assignment]
```

I think its occuring because I originally set `src: str` in `PreprocessConfig`, but then override it to `src: Callable` in `PreprocessConfigResolved`. I wasn't able to find an obvious simple fix to this, except to not use the approach of having the `Resolved` config's inherit from the original configs. Not doing this would be fine, but just a fair bit more repetition...

Reply:
Replaced this with `# type: ignore[assignment]`. Not sure if there's a better approach...